src/item.h

```diff
@@ -1367,10 +1367,16 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
          * Summed range value of a gun, including values from mods. Returns 0 on non-gun items.
          */
         int gun_range( bool with_ammo = true ) const;
+
+        /** Get handling bonus (a measure of how effective gun + attached mods control recoil) */
```

Comment:
Could the effect be explained in few words?
For example (from what I can tell correct): "Actual recoil applied to gun is divided by handling" or something like that.